worker-sandbox/src/test/export_durable_object.rs

```diff
@@ -68,6 +68,18 @@ impl DurableObject for MyClass {
                         "Didn't get the right HashMap<String, i32> using get_multiple"
                     );
 
+                    {
```

Comment:
This code isn't actually hit via the testing suite sadly. I went ahead and added a commit adding testing for it using the new vitest + miniflare pattern.

Reply:
We should rewrite all the durable object tests in the future, but this'll do for now.